docs/pages/x/api/data-grid/data-grid-premium.json

```diff
@@ -82,6 +82,7 @@
         "description": "{ clipboardPaste?: bool, columnGrouping?: bool, lazyLoading?: bool, warnIfFocusStateIsNotSynced?: bool }"
       }
     },
+    "filterDebounceMs": { "type": { "name": "number" }, "default": "50" },
```

Comment:
Should we decrease the default value more gradually, e.g. use 200 or 100 in in v6 and change it to 50 in v7?
My concern is that the UX might degrade on large datasets.
What do you think?